tests/test_extensions/test_ext_apidoc.py

```diff
@@ -356,10 +347,9 @@ def test_toc_all_references_should_exist_pep420_disabled(make_app, apidoc):
         'TOC:\n{}'.format(", ".join(missing_files), toc)
 
 
-def extract_toc(path):
+def extract_toc(path: Path):
```

Comment:
```suggestion
def extract_toc(path: Path) -> None:
```

(You know I don't like half-typed defs)